static/templates/settings/organization-profile-admin.handlebars

```diff
@@ -13,7 +13,7 @@
             </div>
 
             <div class="organization-settings-parent">
-                <div class="input-group admin-realm">
+                <div class="input-group admin-  ">
```

Comment:
`admin-` doesn't seems like a good class name,  idk if it is intentional but changing it would to something more appropriate.